### ISSUE TITLE
Ability to add the exact time to message overview

### ISSUE THREAD

**Issue by GutHib** [thunderbird/thunderbird-android#7754]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.801

### Problem you are trying to solve

On some occasions, you will be getting several messages in a day which have the exact same topic title. With only the date as a distinguisher, it is sometimes very hard to identify the mail you're looking for.

### Suggested solution

It would be great if the overview could always show the time the mail was received, regardless of the date. Similarly, I'd prefer K-9 to consistently show the date instead of displaying "Mo" for a mail that was received six days ago (I'm sure this was done with the best of intentions, but it really makes things more complicated).

Of course, displaying this information could be optional.

**Comment by jayb-g** [comment]
Should be the default. But at least having an option to shown time would be good too.

**Comment by foxacid** [comment]
I agree, the format of time and date in the overview should be a user option.

**Comment by kewisch** [comment]
I think this is interesting to explore, the sticky headers idea sounds like good idea. I'm confirming this, but can't promise any immediate action. For someone wanting to pick this up let's have a chat via Matrix or on this issue, maybe with the design folks included, to see what is the best option.

**Comment by cketti** [comment]
Does this need a separate issue or is this covered by #3987?
